.travis.yml

```diff
@@ -1,12 +1,4 @@
 sudo: false
 language: node_js
 node_js:
-    - "0.10"
-    - "0.11"
-    - "0.12"
-    - "iojs"
-matrix:
-    allow_failures:
-        - node_js: "0.11"
-        - node_js: "0.12"
-        - node_js: "iojs"
+    - "4"
```

Comment:
Not sure how to test against node.js 0.x. Can we tell travis which jsdom to use for testing on different platform?